Vishnu_Sahasranamam - English.tex

```diff
@@ -92,7 +92,21 @@
 % yadbhaktyā puṁḍarīkākṣaṁ stavairarcennaraḥ sadā || 14 ||
 \slokash{Esha me sarva \x{Dh}{•}arm\y{a}{--}n\y{a}{--}m}{\x{Dh}{•}ar\y{mo}{---} - A\x{dh}{•}ikata\y{mo}{---} mataha}
 {Yat - \x{Bh}{•}akty\y{a}{--} Pundar\y{ee}{---}k\y{a}{--}ksham}{Stavaih arch\y{e}{--}n narah sad\y{a}{--} (Stavair/arch\y{e}{--}n/narah)}
+
 % paramaṁ yō mahattejaḥ paramaṁ yō mahattapaḥ |
 % paramaṁ yō mahadbrahma paramaṁ yaḥ parāyaṇam || 15 ||
+\slokash{Paramam Y\y{o}{-} mahat-t\y{e}{-}jaha}
```

Comment:
it will have to be \y{o}{--}. because two dashes are required to have a full dashed line above the o. same with all the characters that has a dash above them. 
for double characters, such as bh, it needs three dashes. something like this \y{bh}{---}

\y adds accent above the character, and \x adds accent below the character.

Reply:
Hi Sriram, Thank You for reviewing the changes and sharing your feedback.